cmd/user-agent/src/store/modules/options.js

```diff
@@ -50,6 +51,31 @@ export default {
                     })
 
                 agentOpts['http-resolver-url'] = agentOpts['http-resolver-url'].split(',')
+
+                const userInfoURL = agentOpts["edge-agent-server"] + "/oidc/userinfo"
+
+                console.log("User info URL is: " + userInfoURL)
+
+                const client = axios.create({
+                    withCredentials: true
+                })
+
+                await client.get(userInfoURL)
+                    .then(resp => {
+                        const edvVaultURL = resp.data.bootstrap.edvVaultURL
+
+                        console.log("User EDV Vault URL is: " + edvVaultURL)
+
+                        const edvVaultID = edvVaultURL.substring(edvVaultURL.lastIndexOf('/')+1)
+
+                        console.log("User EDV Vault ID is: " + edvVaultID)
```

Comment:
@DRK3 let's remove vaultID.. only vaultURL is needed in hub-auth.

hub-auth should have only 1 value for edv vault (and only 1 value for the keystore). 

cc @llorllale , @aholovko @rolsonquadras @fqutishat

Reply:
This `substring` code just extracts the vault ID from the vault URL. The vault ID is used by the Aries EDV REST provider. We could potentially make hub-auth just store the vault ID instead of the full URL. I don't think it makes much difference either way, though.